data/services.json

```diff
@@ -157,6 +174,23 @@
         "supportEmail": "support@ledger.com",
         "website": "https://ledger.com"
       }
+    },
+    {
+      "f_type": "Service",
+      "f_vsn": "1.0.0",
+      "type": "authn",
+      "method": "EXT/RPC",
+      "uid": "flipper-developer#authn",
+      "endpoint": "chrome-extension://nkjhfapihbllbhhneonipcinhdlcmfgn/index.html",
```

Comment:
```suggestion
      "endpoint": "chrome-extension://nkjhfapihbllbhhneonipcinhdlcmfgn/index.html",
      "optIn": true,
```

Reply:
As Flipper does not currently support a user-signature service, you will need to specify "optIn": true.
dApps will be able to specify inclusion via fcl config if they do not require user-signature support.